DevSecOps-WoW.md

```diff
@@ -0,0 +1,25 @@
+# CDC ReportStream DevSecOps Ways of Working
+
+## Team Structure
+
+- The CDC DevSecOps Team includes multiple members, although no person is assigned to a specific Application Team.
+- The CDC DevSecOps Team operates on more of a 'help desk' model where they receive tickets and internally triage/prioritize their work.
+
+## Requesting DevSecOps Support
+
+- Quick questions that take <5 min to answer can be asked in Slack and answered by anyone with knowledge, but any issues that need investigation or that may take longer than 5 minutes to gather context and resolve should be opened in ticket form using the [DevSecOps-issue.md](../ISSUE_TEMPLATE/DevSecOps-issue.md) ticket template.
+- Please avoid using DMs to communicate directly with the DevSecOps Team - instead post communication in the shared #prime-devops channel and tag the appropriate individual(s).
+- Apps Teams will request DevSecOps support via the  [DevSecOps-issue.md](../ISSUE_TEMPLATE/DevSecOps-issue.md)  Github ticket template.  The template will automatically add appropriate labels and assign the ticket to the DevSecOps Team.
+- The DevSecOps Team will track all DevSecOps issues on the ReportStream board that are tagged with the `DevSecOps` and `reportstream` labels.
+- When new issues are created using the DevSecOps issue template, the DevSecOps Team will receive an automatic Slack notification in a separate DevSecOps alert channel (TBD).
+
+## Workflow Management/Task Prioritization
+
+- The DevSecOps Team will operate using a Kanban approach and stay within established Work in Progress (WIP) limits.
+- While the DevSecOps Team won't be embedded with specific Apps Teams, they will still operate within the same sprint cadence as the Apps Teams.
+- DevSecOps will review new tickets as they come in and prioritize them.
+- Every 2 weeks on Wednesday, a DevSecOps Team representative will attend a Scrum of Scrums with the Apps Tech Leads (and TBD) to review the status of in progress items, flag blockers, and discuss prioritization of new issues.  Patrick will give ultimate guidance on what issues need to take priority over others in scenarios, wherever it is unclear.
+- DevSecOps Team members are not required to attend Apps Team stand ups, sprint planning, refinement, or retro meetings given that they may be doing work across multiple apps at a given time. That said, they will join Apps Team stand ups at their discretion if they need to communicate with the Team, and/or seek clarifications or are requested to join to provide input. They will also pair with Tech Leads to flesh out draft issues, when their input is needed.
+- (Eventually) DevSecOps Team members will not factor into app Team velocity calculations.
+- The DevSecOps Team will conduct their own internal stand ups for 10-15 minutes and other recurring Agile ceremonies.
+- DevSecOps Team members will communicate status of their tickets with the respective App Tech Leads.
```

Comment:
New item proposal at the bottom:

- DevSec0ps team shall participate in, and help inform, quarterly program planning exercises

cc @chris-kuryak

Reply:
Agreed.  Adding.